pkg/cloudprovider/cloudapi/aws/aws_account_mgmt.go

```diff
@@ -44,7 +44,7 @@ func setAccountCredentials(client client.Client, credentials interface{}) (inter
 
 	awsConfig := &awsAccountConfig{
 		AwsAccountCredential: *accCred,
-		region:               strings.TrimSpace(awsProviderConfig.Region),
+		region:               strings.TrimSpace(awsProviderConfig.Region[0]),
```

Comment:
Add a comment why we are using index 0 for now

Reply:
added